elements/pfe-navigation/src/pfe-navigation.js

```diff
@@ -358,6 +373,25 @@ class PfeNavigation extends PFElement {
     this._activeNavigationItems.map(item => item.close());
     this.overlay = false;
   }
+
+  /**
+   * Set a global CSS variable reporting the height of this navigation item.
+   * Used to position sticky subnavigation items under this.
+   *
+   * The name of the global CSS variable is `--pfe-navigation--ReportedHeight`.
+   * If this nav has an `id` attribute, the id will be appended to the variable
+   * name to distinguish it from other pfe-navigation items on the page
+   * (unlikely, but imagine a demo page with 20 example pfe-navigation elements
+   * in different states, and one genuine pfe-navigation element at the top
+   * used for actual navigation).
+   */
+  _reportHeight() {
+    const cssVarName =
+      `--pfe-navigation--${this.id ? `${this.id}--` : ""}Height--actual`;
```

Comment:
```suggestion
      `--pfe-navigation__${this.id ? `${this.id}--` : ""}Height--actual`;
```

Reply:
@castastrophe replaced `--` with `__`

Reply:
@castastrophe @mwcz looking at this again, if there is no ID, the resulting custom property will look like this:
`--pfe-navigation__Height--actual` 

Is that correct or should it be `--pfe-navigation--Height--actual` in which case we'd need to move the `__` into the ternary?